spec/lib/spree/active_shipping/bogus_carrier.rb

```diff
@@ -8,7 +8,7 @@ def name
       end
 
       def find_rates(origin, destination, packages, options = {})
-        RateResponse.new(true, "Bogus rate response.")
+        ::ActiveShipping::RateResponse.new(true, "Bogus response")
```

Comment:
the ::ActiveShipping is not needed after all, since it's derived from ::ActiveShipping::Carrier already